test/FunctionalTests/CoreCLRTests/PublishWithDebugTest_CoreCLR.cs

```diff
@@ -1,56 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System.IO;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Server.IntegrationTesting;
-using Microsoft.Extensions.Logging.Testing;
-using Xunit;
-using Xunit.Abstractions;
-
-namespace FunctionalTests
-{
-    public class PublishWithDebugTest_CoreCLR :
```

Comment:
This is a bit problematic. We use the project's configuration to determine the location of the tasks and tools e.g. `bin\$(Configuration)\net461`, essentially the same thing we do with the Razor Sdk. In release builds, there isn't a `Debug` build of the tools and tasks and causes this to fail. Fixing this is non-trivial but also not super important since we don't vary the build targets based on the project's configuration.